R/S3.R

```diff
@@ -70,9 +70,10 @@ pander <- function(x = NULL, ...) {
     ## save current knitr.auto.asis option
     kaao <- panderOptions('knitr.auto.asis')
 
-    if (isTRUE(panderOptions('knitr.auto.asis')) &&
-        isTRUE(getOption('knitr.in.progress')) &&
-        requireNamespace('knitr', quietly = TRUE)) {
+    if (isTRUE(getOption('jupyter.in_kernel')) |
```

Comment:
It is necessary as `pander` does not return anything by design (outside of `knitr` or Jupyter).